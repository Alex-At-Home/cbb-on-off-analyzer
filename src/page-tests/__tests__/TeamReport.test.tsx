import React from 'react';
import TeamReportPage from '../../pages/TeamReport';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe("TeamReportPage", () => {
  test("TeamReportPage - should create snapshot", () => {
    const wrapper = shallow(<TeamReportPage/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
