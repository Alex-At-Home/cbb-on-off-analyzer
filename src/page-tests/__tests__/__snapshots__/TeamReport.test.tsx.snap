// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamReportPage TeamReportPage - should create snapshot 1`] = `
<Container
  className="medium_screen"
  fluid={false}
>
  <SiteModeDropdown />
  <Row
    className="mt-2"
    noGutters={false}
  >
    <Col
      className="text-center"
      xs={12}
    >
      <LandingPageIcon />
      <h3>
        Team On/Off Report Tool
      </h3>
    </Col>
  </Row>
  <Row
    noGutters={false}
  >
    <HeaderBar
      common={Object {}}
      thisPage="report-"
    />
  </Row>
  <Row
    noGutters={false}
  >
    <GenericCollapsibleCard
      minimizeMargin={false}
      screenSize="medium_screen"
      summary="On/Off Report: 2024/25  (M): query:'' (sort:desc:off_poss, filter:'', show:[])"
      title="Team Report Filter"
    >
      <TeamReportFilter
        onChangeState={[Function]}
        onStats={[Function]}
        startingState={Object {}}
      />
    </GenericCollapsibleCard>
  </Row>
  <Row
    noGutters={false}
  >
    <GenericCollapsibleCard
      helpLink="https://hoop-explorer.blogspot.com/2020/03/understanding-team-report-onoff-page.html"
      minimizeMargin={true}
      screenSize="medium_screen"
      title="Team Analysis"
    >
      <TeamReportStatsTable
        dataEvent={
          Object {
            "lineupStats": Object {},
            "rosterStats": Object {
              "baseline": Array [],
              "global": Array [],
              "off": Array [],
              "on": Array [],
              "other": Array [],
            },
            "teamStats": Object {
              "baseline": Object {
                "doc_count": 0,
              },
              "global": Object {
                "doc_count": 0,
              },
              "off": Object {
                "doc_count": 0,
              },
              "on": Object {
                "doc_count": 0,
              },
            },
          }
        }
        onChangeState={[Function]}
        startingState={Object {}}
      />
    </GenericCollapsibleCard>
  </Row>
  <Footer
    server="server"
  />
</Container>
`;
